src/ImageSharp/Formats/Jpeg/JpegDecoderResizeMode.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+namespace SixLabors.ImageSharp.Formats.Jpeg
+{
+    /// <summary>
+    /// Provides enumeration for resize modes taken during decoding.
+    /// Applicable only when <see cref="DecoderOptions.TargetSize"/> has a value.
+    /// </summary>
+    public enum JpegDecoderResizeMode
+    {
+        /// <summary>
+        /// Both <see cref="IdctOnly"/> and <see cref="ScaleOnly"/>.
+        /// </summary>
+        Combined,
+
+        /// <summary>
+        /// IDCT-only to nearest block scale.
```

Comment:
I suppose it's worth mentioning that quality of this mode is close to the 'Box' resampling/scaling.